ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.view.js

```diff
@@ -249,6 +252,21 @@ module.exports = Marionette.LayoutView.extend({
       homeBoundingBox !== undefined ? homeBoundingBox : defaultHomeBoundingBox
     )
   },
+  addPanZoom: function() {
+    const self = this
+    const PanZoomView = Marionette.ItemView.extend({
+      template() {
+        return (
+          <Gazetteer setState={({ polygon }) => self.map.doPanZoom(polygon)} />
+        )
+      },
+    })
+    this.$el
```

Comment:
Why isn't this added to the [map.hbs](https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.hbs) file, and the region added [here](https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.view.js#L139) instead?